### ISSUE TITLE
How to view the result of NewFastFeatureDetector.Detect(img)?

### ISSUE THREAD

**Issue by stevef1uk** [hybridgroup/gocv#233]
Sorry, this is a question & not an issue per se although it may be gocv.drawKeypoints doesn't exist yet so this could be a request for that to be implemented. 

I would liked to see the result of NewFastFeatureDetector. I am using gocv to look for features in an image following this documentation:  https://docs.opencv.org/3.0-beta/doc/py_tutorials/py_feature2d/py_fast/py_fast.html

My summarised code:

    img := gocv.IMRead("filename.jpg", gocv.IMReadColor)
    ffd := gocv.NewFastFeatureDetector()
    kp := ffd.Detect(img)

What I would like to now do is:

gocv.DrawKeypoints(img, kp, color.RGBA{255, 0, 0, 0})

I am pretty new to OpenCV and gocv so is there a simple alternative approach I could take?

Regards

Steve

**Comment by deadprogram** [comment]
Hello @stevef1uk you are correct that `drawKeypoints()` has not yet been implemented.

A PR that added this would be most gratefully accepted!

**Comment by milosgajdos83** [comment]
@stevef1uk I've opened a PR for this [here](https://github.com/hybridgroup/gocv/pull/242)

If it gets merged you can test it using this code:

```go
package main

import (
	"fmt"
	"image/color"
	"os"
	"path/filepath"

	"gocv.io/x/gocv"
)

func main() {
	simplePath := filepath.Join("simple.jpg")
	// read image
	simple := gocv.IMRead(simplePath, gocv.IMReadColor)
	if simple.Empty() {
		fmt.Printf("Failed to read image: %s\n", simplePath)
		os.Exit(1)
	}
	ffd := gocv.NewFastFeatureDetector()
	kp := ffd.Detect(simple)

	simpleKP := gocv.NewMat()
	gocv.DrawKeyPoints(simple, kp, &simpleKP, color.RGBA{255, 0, 0, 0}, gocv.DrawDefault)

	outPath := filepath.Join("simple_keypoints.jpg")
	if ok := gocv.IMWrite(outPath, simpleKP); !ok {
		fmt.Printf("Failed to write image: %s\n")
		os.Exit(1)
	}
}
```

`simple.jpg`
![simple](https://user-images.githubusercontent.com/1392526/41804941-96eac250-7697-11e8-97a0-60e854e4a88d.jpg)

`simple_keypoints.jpg`:
![simple_keypoints](https://user-images.githubusercontent.com/1392526/41804948-b3067830-7697-11e8-90ef-2b99aaf22715.jpg)

**Comment by deadprogram** [comment]
This code is now released as part of v0.14.0 so now closing. Thank you!
